addons/account/models/account_move.py

```diff
@@ -413,7 +413,7 @@ def _get_counterpart(self):
         help="This field is used for payable and receivable journal entries. You can put the limit date for the payment of this line.")
     date = fields.Date(related='move_id.date', string='Date', required=True, index=True, default=fields.Date.context_today, store=True, copy=False)
     analytic_line_ids = fields.One2many('account.analytic.line', 'move_id', string='Analytic lines', oldname="analytic_lines")
-    tax_ids = fields.Many2many('account.tax', string='Taxes')
+    tax_ids = fields.Many2many('account.tax', string='Taxes', domain=['|', ('active', '=', False), ('active', '=', True)])
```

Comment:
and not on the tax_line_id field, one line below?

Reply:
@qdp-odoo m2o display inactive objects correctly, which is not the case for m2m, that's why I did it for tax_ids only.

Reply:
ok, good to know :-)

thanks for the responsiveness